app/Providers/LoggerProvider.php

```diff
@@ -12,25 +12,30 @@
 
 namespace Vokuro\Providers;
 
+use Phalcon\Config;
+use Phalcon\Logger\Adapter\Stream as FileLogger;
+use Phalcon\Logger\Formatter\Line as FormatterLine;
+use function Vokuro\config;
+
+/**
+ * Logger service
+ */
 class LoggerProvider extends AbstractProvider
```

Comment:
@Jeckerson How about service provider interface 
https://github.com/phalcon/cphalcon/blob/3d76cdaee5898d9566f59b8996c4ddbe4b29f729/phalcon/Di/ServiceProviderInterface.zep#L39-L45

Reply:
Oh nice, didn't saw that!

Reply:
af80a57